frontend/src/metabase-lib/filter.unit.spec.ts

```diff
@@ -1,3 +1,5 @@
+import moment from "moment-timezone"; // eslint-disable-line no-restricted-imports -- deprecated usage
```

Comment:
we shouldn't be using `moment-timezone` as it's deprecated

Reply:
We switched to dayjs, but migrating the **existing** usages of moment to it should be a task in itself.

Reply:
true, but we should try not to introduce new usages of deprecated moment-timezone if possible

Reply:
How would you test something that directly depends on the `moment`?

Reply:
we probably need to add more specific assert

Reply:
@nemanjaglumac actually I missed that we use moment for formatting under the hood

```
function serializeDate(date: Date): string {
  return moment(date).format(DATE_FORMAT);
}
```

so my apologies

Reply:
No worries.